### DISCUSSION TITLE
Welcome to Tasmota Discussions!

### DISCUSSION THREAD

**Discussion by csowada** [csowada/Tasmota#2] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by patience4711** [comment]
Thanks for your excelent work.
I tried to compile in Arduino IDE but that failed. I had to add a     #define USE_AP_SYSTEMS    in  my_user_config.h.  Maybe it was forgotten.

**Reply by csowada** [reply]
There is an environment for the aps system. You can select it in Visual Studi Code with platformIO or set it as pio command parameter.
